pkg/controllers/custompackage/test/resources/customPackages/testDir/app2.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: my-app2
+  namespace: argocd
+spec:
+  destination:
+    namespace: my-app2
+    server: "https://kubernetes.default.svc"
+  sources:
+    - repoURL: cnoe://busybox
+      targetRevision: HEAD
+      path: "."
```

Comment:
Sorry if I missed something but this test case only includes one source and not (as title mentions) several sources !

Reply:
Yeah it's meant to represent when the sources field is used. Which is not exactly the same as what is described in the title. I'll update the test.